### DISCUSSION TITLE
All bypasses not working

### DISCUSSION THREAD

**Discussion by dekid69** [zek-c/Securly-Kill-V111#83] [General]
I'm trying to find a bypass or how to make a bypass for securely for the pass 4 months ever since my school made a patch.

My school's securely uses self hsoted extension method instead using the chromestore
the extension is is knneimmpeamikmjijmnhjmmobddleohj
the version is 2.98.61
the source of installation: Not from Chrome Web Store.
chromebook version: 1.20

**Comment by zek-c** [comment]
making a securely exploit isn't easy. i'll give you a few approaches tho:
block cookies and all connections from securly 

corrupt the extension (either by bugged code or browser based cache)

find a way to make a browser that the extension doesn't influence

**Comment by Slothisslow09** [comment]
but how do we corrup them

**Comment by zek-c** [comment]
you gotta find out 🙄

**Comment by Slothisslow09** [comment]
tell us how? please

**Comment by zek-c** [comment]
i can't because i don't know. if i did id already make an exploit. you have to find these things, im not the securly specialist

**Comment by dekid69** [comment]
Didn’t you make that exploit?

**Comment by zek-c** [comment]
yes but you can't reuse the same method. new stuff can always be found, i'm always on the lookout. for example, securly unblocks every domain with "securly.com" in it. even if it's fnsecurly.com it will allow it (i think) or maybe its host name

**Comment by dekid69** [comment]
@zek-c It mght work to reuse the same method but the problem is that the extension is being used as "self hosted" instead using the chromestore.

**Comment by dragon731012** [comment]
Bro zek bypassi made it not you

**Comment by dragon731012** [comment]
And is chrome://network blocked for you

**Comment by dekid69** [comment]
@dragon731012 no crap that zek made the bypass. and chrome://network isn't blocked for me

**Comment by dragon731012** [comment]
Than use cauDNS

**Comment by dragon731012** [comment]
and he didn’t make it bro it even says made by bypassi on it 💀

**Comment by dekid69** [comment]
@dragon731012 
cauDNS dosent work for me, it works for importing but it doesn't do anything

**Reply by Mattisgoat** [reply]
same

**Comment by dragon731012** [comment]
I’ll help 
1. Go to https://edu.triutama.com
2. Do the instructions 
3. Check network settings if it changed name servers
4. If it did, chrome://restart
5. If it didn’t, you did something wrong

**Reply by Mattisgoat** [reply]
when i do it it is just an infinite reload but when I do It at home it works can you fix that?

**Comment by dekid69** [comment]
@dragon731012 
when i follow the instructions and did the chrome://restart the manage icon still shows on the wifi
when I import the network before the restart, it made the icon gone.

**Comment by dekid69** [comment]
and when i try to search anything up it would be blocked

**Comment by dekid69** [comment]
although custom name works even tho its managed

**Comment by dekid69** [comment]
@zek-c why you closing the issue

**Comment by zek-c** [comment]
because 
1. you can't get an easy exploit to work
2. there are no other solutions
3. its dead (1 day inactivity)

**Comment by Slothisslow09** [comment]
fdjsakl

**Comment by albin83** [comment]
Tbh i just want links  but can you make something that only works with a proxy like a ultra violet one ?
Oh and if you have. A dc tell me

**Comment by Gigibamboo** [comment]
@dragon731012 Will it Unblokc if you use an un-managed wifi's Ip adress, I'll write what happened after I test.

**Comment by albin83** [comment]
> @dragon731012 Will it Unblokc if you use an un-managed wifi's Ip adress, I'll write what happened after I test.

If the website is blocked wont happen but if the website is banned in the school township it should work  bc when it’s banned in the township it means it’s only banned in school grounds

**Comment by zek-c** [comment]
the way a wifi blocker works is every bit of https trafiic is run through their servers and if its not allowed it blocks it. if you can use a VPN (or proxy) that isnt blocked by the wifi servers then the http traffic will be sent through VPN servers instead of the wifi blocker servers.

**Comment by Slothisslow09** [comment]
am 

> > @dragon731012 Will it Unblokc if you use an un-managed wifi's Ip adress, I'll write what happened after I test.
> 
> If the website is blocked wont happen but if the website is banned in the school township it should work bc when it’s banned in the township it means it’s only banned in school grounds

am at my house still blocked

**Comment by dragon731012** [comment]
Go to ur home WiFi's settings and set servers to 167.86.91.171

**Comment by Slothisslow09** [comment]
> Go to ur home WiFi's settings and set servers to 167.86.91.171
i cant change my homes wifi settings

**Comment by dragon731012** [comment]
Why not

**Comment by dekid69** [comment]
My school just have new update to my Chromebook
now they disabled the settings, I cant even access the chrome://network anymore

**Comment by zek-c** [comment]
is there any issue left to fix/solve or can this be closed?

**Comment by Kassie-chan** [comment]
My school has blocked the network and the settings on my Chromebook. My school uses securly as an extension and at the school wifi you can't even use cauDNS but on my home wifi I was able to get it to work. Is it possible for this bypass to run out and quit working, and if you powerbutton+escape+reload your Chromebook, will it stop working?

**Comment by local-slav** [comment]
well im not saying its over but it was a good run

**Comment by Slothisslow09** [comment]
i have one way arround it but yeah its not that common to see

**Comment by Slothisslow09** [comment]
heck i am on discord rn

**Comment by local-slav** [comment]
on god how

**Comment by Slothisslow09** [comment]
i have my ways

**Comment by Slothisslow09** [comment]
tehe

**Comment by dekid69** [comment]
☹️

**Comment by Slothisslow09** [comment]
neh heh

**Comment by Slothisslow09** [comment]
do u use discord?

**Comment by Mattisgoat** [comment]
I do can you dm me how to do it, cau dns works at my home network but not at school.

**Comment by Slothisslow09** [comment]
whats ur discord?

**Comment by Mattisgoat** [comment]
IShowMatt

**Comment by local-slav** [comment]
just change your ip it works for me now

**Comment by Slothisslow09** [comment]
idk how and what ip works

**Comment by zek-c** [comment]
go to network > your wifi> advanced > auto-set Ip or something like that> off> set IP to something else

**Reply by Supermon7** [reply]
the thing is it says my organization has blocked thye Chrome://network and chrome://network#state. So changing my ip wont do anything becuase organization blocked it not securly. Is there a way to bypass your organization?

**Reply by zek-c** [reply]
you don't need to go to chrome://network to change your ip... it's in the settings app

**Reply by Slothisslow09** [reply]
yeah I got it to work awhile ago

**Comment by Slothisslow09** [comment]
ok

**Comment by Slothisslow09** [comment]
idk how but i some how found a bypass but i cant find out how i did the by pass

**Comment by zek-c** [comment]
closed because there is no real problem anymore and every1 just chats here

**Comment by Slothisslow09** [comment]
but I like the chat

**Comment by dekid69** [comment]
Same

**Comment by Slothisslow09** [comment]
lol

**Comment by zek-c** [comment]
start a discussion then

**Comment by dekid69** [comment]
I no no wanna ;(

**Comment by zek-c** [comment]
banned!

**Comment by Slothisslow09** [comment]
doh

**Comment by Slothisslow09** [comment]
do y'all have any more server IP in the networks tab I can use? because my current one is down

**Reply by zek-c** [reply]
https://discord.gg/unblock
annoy them here, i dont like them anyways

**Reply by Slothisslow09** [reply]
ummm that's blocked

**Reply by Slothisslow09** [reply]
oh its a server

**Reply by Slothisslow09** [reply]
no I mean things I can change in my computers settings with the network know any IPS?

**Comment by Slothisslow09** [comment]
I don't think the IP change in network thing works anymore

**Reply by winntterr** [reply]
It doesn't
Pretty much nothing does anymore

**Reply by Slothisslow09** [reply]
its up again

**Comment by winntterr** [comment]
To be honest, the exploit community is looking dry. I remember 3 years ago wondering which would happen first, we run out of exploits or we find a permanent way. Before javascript was disabled (bookmarklets), we had a very very good method that some of my friends still have access to now. However, there is nothing new anymore. So here's what I feel the community should focus on (if anybody cares enough to listen). We should all try to find a way to re-enable bookmarklets, whether that be through a custom made app or changing source code.

TLDR: We as a community need to find a way to re-enable bookmarklets, as that's the key to fixing all this. The exploiting community is too dry and we need to do something about it.

**Reply by winntterr** [reply]
The bookmarklet mentioned is posted in @zek-c repository, it was very very good and I miss having an unblocked chromebook

**Reply by winntterr** [reply]
Also if anybody finds anything, please let me know

**Reply by Slothisslow09** [reply]
yeah nothing so far
